packages/superset-ui-demo/storybook/stories/plugins/plugin-chart-echarts/Stories.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { SuperChart, getChartTransformPropsRegistry } from '@superset-ui/chart';
+import { number, radios, select, withKnobs } from '@storybook/addon-knobs';
+import { EchartsTimeseriesChartPlugin } from '@superset-ui/plugin-chart-echarts';
+import transformProps from '@superset-ui/plugin-chart-echarts/lib/Timeseries/transformProps';
+import { withResizableChartDemo } from '../../../shared/components/ResizableChartDemo';
+import data from './data';
+
+new EchartsTimeseriesChartPlugin().configure({ key: 'echarts-timeseries' }).register();
+
+getChartTransformPropsRegistry().registerValue('echarts-timeseries', transformProps);
+
+export default {
+  title: 'Chart Plugins|plugin-chart-echarts',
+  decorators: [withKnobs, withResizableChartDemo],
+};
+
+export const Timeseries = ({ width, height }) => {
+  const enableForecast = radios('Enable forecast', { Yes: 'yes', No: 'no' }, 'yes');
+  const forecastEnabled = enableForecast === 'yes';
+  const queryData = data
+    .map(row =>
+      forecastEnabled
+        ? row
+        : {
+            __timestamp: row.__timestamp,
+            Boston: row.Boston,
+            California: row.California,
+            WestTexNewMexico: row.WestTexNewMexico,
+          },
+    )
+    .filter(row => forecastEnabled || !!row.Boston);
```

Comment:
Here we simulate what the backend would do, i.e. only returning observations if forecasts aren't enabled.